server/konnectors/edf.coffee

```diff
@@ -0,0 +1,1347 @@
+xml2js = require 'xml2js'
+request = require 'request'
+async = require 'async'
+moment = require 'moment'
+cozydb = require 'cozydb'
+File = require '../models/file'
+
+parser = new xml2js.Parser()
+builder = new xml2js.Builder headless: true
+
+# TODO :
+# Move updateOrCreate in a lib
+# multi account
+# multi contract
+
+# Models
```

Comment:
Yes there is a lot of models, very specific : the whole list [here](http://mesinfos.fing.org/cartographies/datapilote/).
But that is just a great news ! There will be a lot of different and usefull data in the Cozy, so Cozy's users will access to a lot of very interesting and usefull apps reusing them ; ) 

About the specificity to this specific vendor : yes, today it's really specific, and sometimes not well named or structured. We could follow some standard when we found them, but we shouldn't loose data. The Cozy arch allows us to have as much specific field has we want on each documents, and we have to use this feature. But that wy I put a docTypeVersion on each document ; to help developpers track document, and the meaning of their fields, even after models updates, and variations on model's fields.

About the contract model: the name _Contract_ sounds very good to me. Put we may found better fields name from weboob data model as you suggest.

**Documentation** : yes, you're right, I will put it there to. I also create a tool to help developers to reuse data : [mesinfosdataplayground](https://github.com/jacquarg/mesinfosdataplayground)